app/models/card.rb

```diff
@@ -1,2 +1,12 @@
+class CompareValidator < ActiveModel::Validator
+  def validate(record)
+    if record.original_text.downcase  == record.translated_text.downcase
+      record.errors[:base] << "Original and Translated texts are equal"
+    end
+  end
+end
+
 class Card < ActiveRecord::Base
+	validates :original_text,:translated_text, presence: true
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Space missing after comma.<br>Tab detected.